crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -334,19 +351,27 @@ where
                 // If there was no start sequence found for this peer, it is likely a new peer
                 // that has just joined the network, start at 0.
                 info!(peer = ?peer_name, "New gossip peer has joined");
+
+                // TODO: How does this interfere with reconfigurations, etc?
```

Comment:
@laura-makdah / @mystenmark I am not certain that starting from zero is robust to a long sequence under reconfiguration. It is worth understanding this better. In particular what happens to the follower sequence after reconfig? Does it start at zero? Should we have instead of a seq a tuple of (epoch, seq)?

Reply:
If we can successfully receive a checkpoint, then we are only interested in receiving sequences after that checkpoint. How to best modify this to account for that will depend on how checkpoints get streamed.